_visualisation/core/stacks.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Stacks
+position: 1.08
+---
+
+You can group multiple Parties into a single Node, called a Stack, to simplify your diagram. Use this judiciously, only when it improves clarity, and when identifying each individual Party isn’t relevant to your point.
+{: .lead}
+
+When you group Parties like this, the Parties are said to be "stacked", and are "in" or "inside" the Stack.
+
+
+## Appearance
+
+![Stack in a BOVS Diagram](/visualisation/diagrams/bovs-core-stacks-appearance.png){: style="width: 50%" }
+
+* Stack Nodes are drawn as a series of overlaid circles.
+  * If possible, show as many circles as there were Nodes originally, up to a maximum of 3 circles.
+  * The circles in the background may be drawn with up to a 60% tint.
+* Label the Stack with the number and type of grouped Parties.
+  * If there is more than 1 type of Party in the Stack, then use the term "Parties".
+* Show an icon on the topmost circle for each type of Party in the Stack.
+  * Avoid Stacks containing multiple types of Party, if you can – this can get messy.
+
+There are two situations in which you are allowed to group Parties, to create either a Chain Stack or a Beneficiary Stack.
+
+
+## Chain Stacks
+
+![Chain Stack in a BOVS Diagram](/visualisation/diagrams/bovs-core-stacks-chain.png){: style="width: 100%" }
```

Comment:
I'm not sure about the third diagram here. 

![image](https://user-images.githubusercontent.com/342624/84676334-4b49cd00-af25-11ea-841b-c4c698abb1b2.png)

I could imagine Person A -> Company X -> Entity B -> Entity C -> Company D collapsing into a diagram like the middle one, but that has '3 parties' in the middle with a Company and Entity icon.

However, combining the Beneficial Owner (Person) in a stack with intermediate interested parties (entities) doesn't feel like something we should be doing or encouraging - as it seems to lead to ambiguous termination of a chain.

Reply:
Agreed

Reply:
OK – the rule can be that you can't collapse the ultimate Beneficiary into a Chain Stack. So Chains always start with a non-Stack. Can a Stack terminate a Chain? e.g. would we ever want to show an ultimate Beneficiary with a line just leading to, say, "5 Entities"?

Reply:
Unlikely. So we don't need to present stacks starting or terminating a chain.